latest/4-Web_Application_Security_Testing/04-Authentication_Testing/02-Testing_for_Default_Credentials.md

```diff
@@ -75,9 +75,11 @@ This type of issue of often difficult to identify from a black-box perspective.
 - [Burp Intruder](https://portswigger.net/burp/documentation/desktop/tools/intruder)
 - [THC Hydra](https://github.com/vanhauser-thc/thc-hydra)
 - [Nikto 2](https://www.cirt.net/nikto2)
+- [Nuclei](https://github.com/projectdiscovery/nuclei)
 
 ## References
 
 - [CIRT](https://cirt.net/passwords)
 - [SecLists Default Passwords](https://github.com/danielmiessler/SecLists/tree/master/Passwords/Default-Credentials)
 - [DefaultCreds-cheat-sheet](https://github.com/ihebski/DefaultCreds-cheat-sheet/blob/main/DefaultCreds-Cheat-Sheet.csv)
+- [Nuclei-Templates](https://github.com/projectdiscovery/nuclei-templates/tree/6b26c63d8f63b2a812a478f14c4c098b485d54b4/http/default-logins)
```

Comment:
This does not seem to be reference material.

Reply:
Yes, that's correct. Is the Nuclei Tool acceptable?

Reply:
Yup the tool is good, and you could even add the template repo as a secondary tool entry. (Like Nuclei as a bull, then templates as a sub-bullet)